win32ss/gdi/eng/pdevobj.c

```diff
@@ -145,8 +155,11 @@ PDEVOBJ_vRelease(
         }
 
         /* Is this the primary one ? */
-        if (ppdev == gppdevPrimary)
-            gppdevPrimary = NULL;
+        if (ppdev == gpmdev->ppdevGlobal)
+            gpmdev->ppdevGlobal = NULL;
+
+        /* Unload display driver */
+        EngUnloadImage(ppdev->pldev);
```

Comment:
Now that I think about it, we don't have anymore an LDEVOBJ_pUnloadDriver ?

Reply:
No, we had EngLoadImageEx (renamed to LDEVOBJ_pLoadDriver), but no trace of EngUnloadImage.
We might rename EngUnloadImage to LDEVOBJ_pUnloadDriver, and make EngUnloadImage a wrapper around it, but I see no interest to do that.